connectors/handlers/views.py

```diff
@@ -111,3 +116,25 @@ def slack_bot_handle_callback_events(request_message: HttpRequest) -> JsonRespon
         else:
             return JsonResponse({'success': False, 'message': f"Received invalid data type: {d_type}"}, status=400)
     return JsonResponse({'success': False, 'message': 'Slack Event Callback  Handling failed'}, status=400)
+
+
+@csrf_exempt
+@api_view(['POST'])
+def pagerduty_handle_incidents(request_message: HttpRequest) -> JsonResponse:
+    try:
+        data = request_message.data
+        handle_pd_incident(data)
+        return JsonResponse({'success': False, 'message': 'pagerduty incident Handling failed'}, status=200)
+    except Exception as e:
+        logger.error(f'Error handling pagerduty incident: {str(e)}')
+        return JsonResponse({'success': False, 'message': f"pagerduty incident Handling failed"}, status=500)
+
+
+@web_api(ExecuteWorkflowRequest)
+def pd_generate_webhook(request_message: ExecuteWorkflowRequest) -> HttpResponse:
```

Comment:
def pagerduty_generate_webhook(request_message: HttpRequest) -> JsonResponse:

Reply:
HttpResponse works. JsonResponse gives an error. 
TypeError: In order to allow non-dict objects to be serialized set the safe parameter to False.